.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+name: TDR Tag and pre deploy
+on:
+  push:
+    branches:
+      - master
+jobs:
+  pre-deploy:
+    uses: nationalarchives/tdr-github-actions/.github/workflows/lambda_build.yml@main
+    with:
+      repo-name: tdr-ecr-scan
+      artifact-name: ecr-scan
+      build-command: |
+        sbt assembly
+    secrets:
+      MANAGEMENT_ACCOUNT: ${{ secrets.MANAGEMENT_ACCOUNT }}
+      WORKFLOW_PAT: ${{ secrets.WORKFLOW_PAT }}
+  deploy:
+    needs: pre-deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: gh workflow run deploy.yml -f environment=intg -f to-deploy=${{ needs.pre-deploy.outputs.next-version }}
```

Comment:
Should the `environment` parameter be set to `intg`?

Reply:
Good spot, the environment parameter shouldn't be there at all, the deploy job doesn't take on as this only gets deployed to management, I've taken it out.